website/docs/guides/core-to-cloud-1.md

```diff
@@ -0,0 +1,244 @@
+icon: 'guides'
+time_to_complete: 'Total estimated time: 3-4 hours'
+hide_table_of_contents: true
+tags: ['Migration','dbt Core','dbt Cloud']
+level: 'Intermediate'
+recently_updated: true
+---
+## Introduction
+
+Moving from dbt Core to dbt Cloud streamlines analytics engineering workflows by allowing teams to develop, test, deploy, and explore data products using a single, fully managed service.
+
+dbt Cloud is the fastest and most reliable way to deploy dbt. Develop, test, deploy, and explore data products using a single, fully managed service. It also supports: 
+  - The [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) or command line with [dbt Cloud CLI](/docs/cloud/cloud-cli-installation) for development
+  - [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) for consistent metrics
+  - Domain ownership of data with multi-project [dbt Mesh](/best-practices/how-we-mesh/mesh-1-intro) setups
+  - [dbt Explorer](/docs/collaborate/explore-projects) for easier data discovery and understanding
+  
+  Learn more about [dbt Cloud features](/docs/cloud/about-cloud/dbt-cloud-features).
+
+dbt Core is an open-source tool that enables data teams to transform data following analytics engineering best practices using a command line interface. It must be self-hosted and maintained.
+
+### What you'll learn
+
+This guide outlines what you need to do in order to move from dbt Core to dbt Cloud and highlights the necessary technical changes:
+
+- [Account setup](https://docs.getdbt.com/guides/core-to-cloud-1?step=3#account-setup): Learn how to create a dbt Cloud account, invite team members, and configure it for your team.
+- [Environment variables](https://docs.getdbt.com/guides/core-to-cloud-1?step=4#environment-variables): Discover how to manage environment variables in dbt Cloud, including their priority.
+- [Data platform setup](https://docs.getdbt.com/guides/core-to-cloud-1?step=5#data-platform-setup): Find out about connecting your data platform to dbt Cloud.
+- [Git setup](https://docs.getdbt.com/guides/core-to-cloud-1?step=6#git-setup): Learn to link your dbt project's Git repository with dbt Cloud.
+- [Developer setup:](https://docs.getdbt.com/guides/core-to-cloud-1?step=7#developer-setup) Understand the setup needed for developing in dbt Cloud.
+- [Orchestration setup](https://docs.getdbt.com/guides/core-to-cloud-1?step=8#orchestration-setup): Learn how to prepare your dbt Cloud environment and jobs for orchestration.
+- [Models configuration](https://docs.getdbt.com/guides/core-to-cloud-1?step=9#models-configuration): Get insights on validating and running your models in dbt Cloud, using either the dbt Cloud IDE or dbt Cloud CLI.
+- [What's next?](https://docs.getdbt.com/guides/core-to-cloud-1?step=10#whats-next): Summarizes key takeaways and introduces what to expect in the following guides.
+
+## Prerequisites
+
+- You have an existing dbt Core project connected to a data platform and Git repository.
```

Comment:
We'll also want to call out that it has to be a version >= 1.0

Reply:
we could! but aren't versions less than 1.0 deprecated anyway?